flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -25,7 +25,7 @@ MINIKUBE_START_RETRIES=3
 MINIKUBE_START_BACKOFF=5
 RESULT_HASH="e682ec6622b5e83f2eb614617d5ab2cf"
 MINIKUBE_VERSION="v1.8.2"
-MINIKUBE_PATH="/usr/local/bin/minikube-$MINIKUBE_VERSION"
+MINIKUBE_PATH=$([ "${OS_TYPE}" != "linux" ] && echo "minikube" || echo "/usr/local/bin/minikube-$MINIKUBE_VERSION")
```

Comment:
Why is this change necessary @wangyang0918?

Reply:
In linux environment, we are downloading the specific minikube version(e.g. v1.8.2) and set the correct `MINIKUBE_PATH`. However, on Mac, we assume that the minikube is pre-installed and we could directly use the `minikube` command to check status or mounting.

Reply:
Currently, when running native K8s related test on Mac, we will get following errors.
```
./test-scripts/common_kubernetes.sh: line 55: /usr/local/bin/minikube-v1.8.2: No such file or directory
****** Please start/stop minikube manually in non-linux environment. ******
```

Reply:
Is `v1.8.2` strictly required? If yes, then we might wanna check on MacOS that `minikube` is actually the correct version. I for example have `1.15.1` installed.

Reply:
Hmm. `v1.8.2` is not strictly required. We are using this version just because it is the latest stable version when we introduce the tests.

Reply:
Then we could maybe change it so that we use whatever version is available and if `minikube` is not available, then we try to download minikube on Linux. On MacOS we fail with a descriptive message. This would be a somewhat consistent behaviour.

Reply:
I remember the azure pipeline updated the provided minikube version silently(a not stable version) and that leads to all the K8s related tests failed. This is why we have pinned the minikube version to `v1.8.2`.

Reply:
Would it make sense to check whether there is a `minikube` version and if whether it is a version which works? In this case we could use it. If not, then the system could try to download and install `minikube` `v1.8.2` if it runs on Linux.